docs/src/main/sphinx/release/release-422.md

```diff
@@ -0,0 +1,49 @@
+# Release 422 (12 Jul 2023)
+
+## General
+
+* Add support for adding nested fields with an `ADD COLUMN` statement. ({issue}`16248`)
+* Improve performance of `INSERT` and `CREATE TABLE AS ... SELECT` queries. ({issue}`18005`)
+* Fix queries hanging when the `task.retry-policy` configuration property is set
+  to `TASK` and other worker node failures occur. ({issue}`18175 `)
+
+## Security
+
+* Add OAuth 2.0 support for validating JWT types. ({issue}`17640`)
+* Fix error when the `http-server.authentication.type` configuration property
+  is set to `oauth2` or `jwt` and the `principal-field` property has a different
+  value. ({issue}`18210`)
+
+## BigQuery connector
+
+* Add support for writing to columns with a `timestamp with time zone` type. ({issue}`17793`)
```

Comment:
Uppercase for types

TIMESTAMP WITH TIME ZONE

Reply:
No, we use lower case for type names in release notes. Also, the type is `timestamp(p) with time zone`

Reply:
ok .. but only in release notes from all I understand